### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by Srilearns**
Added the contact form in the Mobile First Portfolio Website.
Added input fields name, email , text message fields and submit button.
Created styling for the form based on Figma template 1
Implemented button actions

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -222,9 +221,24 @@ <h2>Skills</h2>
...
+             <label for="mail">Email:</label>
+             <input name="user_email" id="mail" type="email" placeholder="loremipsum@gmail.com" required>
+             <label for="msg">Message:</label>
+             <textarea name="user_message" id="msg" placeholder="Write your message here..." maxlength="500" required></textarea>
+             <button type="submit">Get in touch</button>
+            </form>
+            </div>
+            <div class="home-indicator">
+            </div>
+        </section>
```
Comment (Munsa1) [inline index.html:242]: - [ ] Good job creating the contact form, using the correct tags for semantic purposes, and adding all the required validations. However, we have a few minor issues to achieve a great design.

- Kindly align the background image to match the given template

Your design
image

Given design
image


- [ ] Kindly align the text area to align with the input fields and the `Get In Touch` button so as to match the given design. Hint, you can apply Flexbox to it as you have done for the other elements

Your design
image

Given design
image

- [ ] Also add the two background images to the bottom of your design so as to match the given template

Given template
image

Your design
image

File: styles.css [hunk 2]
```diff
@@ -503,3 +515,129 @@ p {
...
+  color: white;
+}
+
+.contact button :active {
+  color: #6070ff;
+}
+
+.contact-mobile-btn:disabled {
+  color: #fff;
+}
```
Comment (Munsa1) [inline styles.css]: - [ ] Good job adding the hover, active and disabled effects. Kindly add the `pressed` effect as well to meet the requirements.
image

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @Srilearns, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Collaboration On Project Achieved :heavy_check_mark: 
Nice UI Design :heavy_check_mark: 
Descriptive PR :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._
### Important 🔴 
Am leaving a comment here due to insufficient files changed

- [ ] Kindly edit your MIT license agreement to reflect your name and app type
image

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Alusp — APPROVED** [review]
Hi @Srilearns, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
congratulations

## Highlights

- [ ] Professional PR Summary :heavy_check_mark: 
- [ ] Good UI design :heavy_check_mark: 
- [ ] Sets HTML Validations correctly :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**